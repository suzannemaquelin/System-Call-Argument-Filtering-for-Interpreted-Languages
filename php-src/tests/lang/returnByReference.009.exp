---> 1. Return a variable by reference -> No warning:
int(1)

---> 2. Return a value by reference -> Warning:

Notice: Only variable references should be returned by reference in %s on line %d
int(1)