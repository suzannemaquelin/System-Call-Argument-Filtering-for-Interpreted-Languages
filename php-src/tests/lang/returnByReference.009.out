Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

---> 1. Return a variable by reference -> No warning:
int(1)

---> 2. Return a value by reference -> Warning:

Notice: Only variable references should be returned by reference in /root/php-src/tests/lang/returnByReference.009.php on line 18
int(1)